### PR TITLE
New version: VeyonSolutions.Veyon version 4.7.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36841)

### CODE REVIEW

File: manifests/v/VeyonSolutions/Veyon/4.7.0/VeyonSolutions.Veyon.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: VeyonSolutions.Veyon
+PackageVersion: 4.7.0
```
Comment (OfficialEsco) [inline manifests/v/VeyonSolutions/Veyon/4.7.0/VeyonSolutions.Veyon.installer.yaml:5]:
```suggestion
PackageVersion: 4.7.0.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/v/VeyonSolutions/Veyon/4.7.0/VeyonSolutions.Veyon.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/veyon/veyon/releases/download/v4.7.0/veyon-4.7.0.0-win32-setup.exe
+  InstallerSha256: F4786A5FC77C3F1C275D799F81584442879D7A7528984DA5CFF70216D92B314D
+- Architecture: x86
+  InstallerUrl: https://github.com/veyon/veyon/releases/download/v4.7.0/veyon-4.7.0.0-win64-setup.exe
+  InstallerSha256: A0B45783F769A01E5FE8A9637A8A7B6255A606AE32A2297D0FAD984A8FA5B620
```
Comment (OfficialEsco) [inline manifests/v/VeyonSolutions/Veyon/4.7.0/VeyonSolutions.Veyon.installer.yaml:21]: also these are flipped

**Comment by vedantmgoyal2009** [comment]
@OfficialEsco Please wait for me to commit/fix issues. I have my exams these days so I am a little less online. After 15th December, I will be more available.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Should be fixed by https://github.com/microsoft/winget-pkgs/pull/36894.

**Comment by vedantmgoyal2009** [comment]
Pipelines having issues again?

**Review by OfficialEsco — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
Forgot to mention, I have added you as a collaborator on my fork so that you can fix _small_ changes and don't have to open a fresh PR running YamlCreate from scratch.

**Pull request merged**